README.md

```diff
@@ -1,87 +1,51 @@
-  updates
+1. Make a copy of `.env.sample` saved as `.env` and provide values for all the environment variables. Visit https://github.com/settings/tokens to generate an access token with 'repo' scope permissions.
+1. `npm install` to install dependencies.
+1. `npm run dev-start` to start an auto-restarting server.
 
-## <a name="installation"></a>Installing the `team-api` server
+Run `npm test` to do a single run of the test suite.
+Run `npm run dev-test` to run the test suite continuously as changes are made.
 
-Install the following if they are not yet present on your system:
+You can run `node scripts/push_hook.js` to send a test payload to your running application.
 
-* [Node.js](https://nodejs.org/) version 0.12.7 or higher;
-  check with `node -v`
-* [Ruby](https://www.ruby-lang.org/) version 2.2.3 or higher;
-  check with `ruby -v`
-* [Git](https://git-scm.com/) version 1.9.1 or higher;
-  check with `git --version`
+### In Production
 
-For Ruby, we strongly recommend using a version manager such as
-[rbenv](https://github.com/sstephenson/rbenv) or [rvm](https://rvm.io/),
-though this is not required.
-
-`rsync` should already be installed on most UNIX-like systems, but the
-`rsyncOpts` [configuration option](#config) may require adjustment,
-particularly on OS X. You may wish to experiment with `rsync` manually to
-determine which options suit you best.
-
-With the correct Node.js, Ruby, and Git versions in place, run the following:
+Instead of using the `.env` file, you may specify normal environment variables:
 
 ```sh
-$ gem install jekyll bundler
-$ npm install -g team-api-server forever
+export GITHUB_USER=<GitHub username>
+export GITHUB_ACCESS_TOKEN=<GitHub access token with 'repo' scope>
+export GITHUB_ORG=<GitHub organization name>
+export DESTINATION_REPO=<GitHub organization name>
+export DESTINATION_PATH=_data/projects
+export DESTINATION_BRANCH=master
+export TARGET_FILE=.about.yml
+export PORT=6000
 ```
-Finally, as the user on the host that will run the server,
-[generate an SSH key to add to your GitHub
-account](https://help.github.com/articles/generating-ssh-keys/). A new key can
-be generated by another team member should you leave the organization.
-
-
-### Run the `team-api` server
-
-After that, run the following to launch the server via
-[Forever](https://www.npmjs.com/package/forever), where `/path/to/` and
-`/usr/local/bin/` are replaced with the appropriate absolute paths:
-
-```sh
-$ forever start -l /path/to/team-api.log -a /usr/local/bin/team-api /path/to/team-api-config.json
-```
-
-You can find the absolute path to `team-api` by running `which team-api`.
 
 ## Contributing
 
 1. Fork the repo (or just clone it if you're an 18F team member)
 2. Create your feature branch (`git checkout -b my-new-feature`)
-3. Make your changes and test them via `npm test` or `gulp test`
-4. Lint your changes with `gulp lint`
+3. Make your changes and test them via `npm test`
+4. Lint your changes
```

Comment:
I don't see that as a pattern generally. Also, I think we're using Code Climate for that kind of thing. I  think maybe I just set that up, but I haven't used Code Climate before: https://codeclimate.com/github/18F/team-api-server